### DISCUSSION TITLE
podman ignores default system connection setting

### DISCUSSION THREAD

**Discussion by peedy2495** [containers/podman#21415] [Q&A] [answered]
Hello,
before I start to create a bug-issue I'm not shure if it is an bug ...
 Host: ubuntu 22.04 using podman 4.9.0 (alt. 4.8.2)
 Client fedora 38 using podman 4.9.0 (alt. 4.8.2)

I configured client system connections via ssh for both existent machine-urls and alternative for my testuser to the remote host.
I'm able to handle the remote host with 'pdman --remote ...', but normal 'podman ...' always points to the local socket althogh the default system connection points to a remote one.

I'm trying to get podman desktop to handle a remote instance.

**Comment by rhatdan** [comment, marked as answer]
Podman default to local mode.

You can change this behavior with containers.conf  with something like:

```
[Engine]
remote=true
```

**Reply by peedy2495** [reply]
Perfect, thank you !

Commandline is working now ...
Now,  podman desktop is another repo for further probs ;-)
